Automattic/jetpack
# Update/migrate stats counts to tanstack

## Proposed changes:

* Get Stats counts using the tanstack query instead of redux
* Add a way to suppress global notice errors for certain queries

### Other information:


## Jetpack product discussion

P2: p1HpG7-rcF-p2

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch via the Jetpack Beta plugin or your local dev environment
2. Make sure you site is **not** connected
3. Go to My Jetpack and make sure the global notice does not show an error
![image](https://github.com/Automattic/jetpack/assets/65001528/86486701-e4cc-4ee4-a736-3416343cc603)
4. Connect your site and make sure the stats load correctly
![image](https://github.com/Automattic/jetpack/assets/65001528/7969b6cb-60b6-4359-ae10-ab6509ac1f9d)
